06-misc_modules.Rmd

```diff
@@ -0,0 +1,15 @@
+# Miscellaneous
```

Comment:
Not sure "misc" is how I'd describe this. I think this fits with billing, no? If not, maybe its own "setup" chapter?

Not a dealbreaker if you feel strongly :)

Reply:
I didn't want to bury it in Billing, since everybody needs it, not just people setting up billing.  Renamed it to "Onboarding"...dunno if we'll have much more to go in that chapter, but at least it indicates what's there.

Reply:
Sounds good!